### PR TITLE
Remove message module

### PR DESCRIPTION
**Pull request by dcousens**
As discussed @weilu.

Can now be found (in isolation) [here](https://github.com/bitcoinjs/bitcoinjs-message).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,8 +91,6 @@ The below examples are implemented as integration tests, they should be very eas
 - [Generate a address and WIF for Litecoin](https://github.com/bitcoin/bitcoinjs-lib/blob/master/test/integration/basic.js#L29)
 - [Import an address via WIF](https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/basic.js#L43)
 - [Create a Transaction](https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/basic.js#L50)
-- [Sign a Bitcoin message](https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/advanced.js#L9)
-- [Verify a Bitcoin message](https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/test/integration/advanced.js#L17)
```
Comment (weilu) [inline README.md]: Shall we keep the bullet points but link to bitcoinjs-message instead?

**Comment by weilu** [comment]
LGTM. Needs rebase though

**Comment by rubensayshi** [comment]
utACK

you intentionally rebased this against https://github.com/bitcoinjs/bitcoinjs-lib/pull/650 I guess?

**Comment by jprichardson** [comment]
🎉  One step closer to being lean and mean!

**Pull request merged**